Document/0x05b-Basic-Security_Testing.md

```diff
@@ -67,225 +94,233 @@ You can get familiar with Magisk reading the official [documentation on GitHub](
 
-##### Network Setup
-
-The available network setup options must be evaluated first. The mobile device used for testing and the machine running the interception proxy must be connected to the same Wi-Fi network. Use either an (existing) access point or create [an ad-hoc wireless network](https://support.portswigger.net/customer/portal/articles/1841150-Mobile%20Set-up_Ad-hoc%20network_OSX.html "Creating an Ad-hoc Wireless Network in OS X").
+###### Root Detection
 
-Once you've configured the network and established a connection between the testing machine and the mobile device, several steps remain.
+An extensive list of root detection methods is presented in the "Testing Anti-Reversing Defenses on Android" chapter.
 
-- The proxy must be [configured to point to the interception proxy](https://support.portswigger.net/customer/portal/articles/1841101-Mobile%20Set-up_Android%20Device.html "Configuring an Android Device to Work With Burp").
-- The [interception proxy's CA certificate must be added to the trusted certificates in the Android device's certificate storage](https://support.portswigger.net/customer/portal/articles/1841102-installing-burp-s-ca-certificate-in-an-android-device "Installing Burp's CA Certificate in an Android Device"). The location of the menu used to store CA certificates may depend on the Android version and Android OEM modifications of the settings menu.
-- Some application (e.g. the [Chrome browser](https://bugs.chromium.org/p/chromium/issues/detail?id=475745 "Chromium Issue 475745")) may show `NET::ERR_CERT_VALIDITY_TOO_LONG` errors, if the leaf certificate happens to have a validity extending a certain time (39 months in case of Chrome). This happens if the default Burp CA certificate is used, since the Burp Suite issues leaf certificates with the same validity as its CA certificate. You can circumvent this by creating your own CA certificate and import it to the Burp Suite, as explained in a [blog post on nviso.be](https://blog.nviso.be/2018/01/31/using-a-custom-root-ca-with-burp-for-inspecting-android-n-traffic/ "Using a custom root CA with Burp for inspecting Android N traffic").
+For a typical mobile app security build, you'll usually want to test a debug build with root detection disabled. If such a build is not available for testing, you can disable root detection in a variety of ways that will be introduced later in this book.
 
-After completing these steps and starting the app, the requests should show up in the interception proxy.
+#### Recommended Tools
+TODO recommend a basic set and installation instructions?
```

Comment:
Let's summarize a few basic tools that we are using throughout the guide, but let's reference to installation manuals and don't copy paste from the tools documentation. let's only put in installation instructions if they are not straight forward or the documentation lacks information. Otherwise we are running out of sync and would need to maintain this also

Reply:
I totally agree with this!